### DISCUSSION TITLE
Amplitude grds

### DISCUSSION THREAD

**Discussion by simonst7** [gmtsar/gmtsar#320] [Q&A] [answered]
Hello, 

I'm trying to get an amplitude map for the reference and secondary image. I'm trying to get this with **amp1.grd** and **amp2.grd**, is that correct?. I know that it's not the GMTSAR purpose, but It could be faster than using other packages for this purpose. I got this from two images before and after the Tonga eruption 2022 (roma colour palette) 

![amp_diff](https://user-images.githubusercontent.com/53209067/165882728-bd2335e8-29ab-493a-8813-e94f54e14b7c.png)

Thank you!

**Comment by Xiaohua-Eric-Xu** [comment]
The amp1.grd and qmp2.grd are filtered results. If you want to get a sharp image, you could try slc2amp.csh.

**Reply by simonst7** [reply]
Thank you @Xiaohua-Eric-Xu , do you know what rng_dec is suitable for Sentinel1 data?. There are some options in slc2amp.csh  such as 1:ERS ENVISAT 2:ALOS 4:TSX . I tried with different numbers but the result doesn't seem to change so much. This is one map in range/az 
 
![20210917_SNT1_VV_raw_rng3](https://user-images.githubusercontent.com/53209067/166090669-3369dcb6-cd07-4cf1-82f9-d6665dd2b971.png)

Also, I wanted to take the opportunity to ask you (maybe it is for another topic) since the site is too small to get the deformation (2.8 km2) and the coherence of the points is very low (maybe also due to the wide sea around). Is there any way to improve this, or maybe different filtering is an option? I see almost all decorrelated, I tried to crop the area for just two bursts in F3, but still remain the decorrelation there. This is the display_amp and correlation maps (F3 - 2 burst) for the area marked in the box.
![amp_corr](https://user-images.githubusercontent.com/53209067/166091096-a5110972-241a-472c-9fcd-f67e7e291c4c.png)

Thank you in advance!

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
Hi @simonst7, I usually use 4 for TOPS data. If you read into the script. It's basically convolving a filter over the region (using `conv`). You can tune the parameters to get best output. I am not particularly sure what's the best way to improve coherence over such area. You could try persistent scatter with `gmtsar2stamps`. Or only pick the one with good coherence and dispersion coefficient (produce from p_scatter). If you want to further cut down the area for processing, you could use `cut_slc`.

**Reply by simonst7** [reply]
Thank you @Xiaohua-Eric-Xu for your time. I'll try to figure out the best way with your advice. 
The `cut_slc` does not cut the area that I want. I used the coregistered PRM files (intf folder), but It didn't crop the correct area. I put the range in this way: rng1/rng2/azimut1/azimut2 is it correct?. 

Thanks

**Reply by Xiaohua-Eric-Xu** [reply]
Hmmm, could you show me an example of what you did? Usually I would just put numbers in to the config file for such purpose. You'll need to figure out your area of interest with SAT_llt2rat first I think.

**Reply by simonst7** [reply]
Hi @Xiaohua-Eric-Xu thank you. now the `cut_slc` worked. I followed this steps:

1. Take the range/azimut of the AOI from topo_ra.grd 
2. Apply `cut_slc` in the coregistered PRM file (with the previous range obtained)
3. Use slc2amp.csh to see the new area (I had to change the name of the new SLC for the old SLC beacuse the new name not worked).
4. Result in the rng/az coordinates
![image](https://user-images.githubusercontent.com/53209067/167770787-a1077f34-f84c-46b7-b7aa-fa1fb9a305ea.png)

Thank you for your help,
Appreciate it
